community/kernel/src/main/java/org/neo4j/kernel/impl/cache/MonitorGc.java

```diff
@@ -53,7 +53,7 @@ public void init() throws Throwable
     @Override
     public void start() throws Throwable
     {
-        monitorGc = new MeasureDoNothing( "GC-Monitor", log, config.get( Configuration.gc_monitor_wait_time ),
+        monitorGc = new MeasureDoNothing( "neo4j.pauseMonitor", log, config.get( Configuration.gc_monitor_wait_time ),
```

Comment:
Hmm, it doesn't belong in this commit, changed in passing. I'm trying to
have the threads use the same naming scheme as in JobScheduler.

I can roll this out and put it in a separate commit, if you want. Imo it's
a small thing tho.
On Jun 9, 2015 8:19 AM, "Zhen Li" <notifications@github.com> wrote:

> In
> community/kernel/src/main/java/org/neo4j/kernel/impl/cache/MonitorGc.java
> <https://github.com/neo4j/neo4j/pull/4791#discussion_r32011108>:
>
> > @@ -53,7 +53,7 @@ public void init() throws Throwable
> >      @Override
> >      public void start() throws Throwable
> >      {
> > -        monitorGc = new MeasureDoNothing( "GC-Monitor", log, config.get( Configuration.gc_monitor_wait_time ),
> > +        monitorGc = new MeasureDoNothing( "neo4j.pauseMonitor", log, config.get( Configuration.gc_monitor_wait_time ),
>
> Is there a special reason to change this name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neo4j/neo4j/pull/4791/files#r32011108>.
>